src/izonereport.cpp

```diff
@@ -89,6 +89,7 @@ main( int _argc, char** argv ) {
     if ( strcmp( myEnv.rodsUserName, PUBLIC_USER_NAME ) != 0 ) {
         status = clientLogin( conn );
         if ( status != 0 ) {
+            printErrorStack(conn->rError);
```

Comment:
Should we include the header for this one?

I'm guessing this compiles alright, but we're adding the header file everywhere else, so might as well. :)

Reply:
Oh you're correct. I missed that.